plan/builders.go

```diff
@@ -71,44 +71,60 @@ type Builder interface {
 	AggregateColumns(input Rel, measures []AggRelMeasure, groupByCols ...int32) (*AggregateRel, error)
+	// Deprecated: Use AggregateExprs(...).ChangeMapping() instead.
 	AggregateExprsRemap(input Rel, remap []int32, measures []AggRelMeasure, groups ...[]expr.Expression) (*AggregateRel, error)
 	AggregateExprs(input Rel, measures []AggRelMeasure, groups ...[]expr.Expression) (*AggregateRel, error)
+	// Deprecated: Use CreateTableAsSelect(...).ChangeMapping() instead.
 	CreateTableAsSelectRemap(input Rel, remap []int32, tableName []string, schema types.NamedStruct) (*NamedTableWriteRel, error)
 	CreateTableAsSelect(input Rel, tableName []string, schema types.NamedStruct) (*NamedTableWriteRel, error)
+	// Deprecated: Use Cross(...).ChangeMapping() instead.
 	CrossRemap(left, right Rel, remap []int32) (*CrossRel, error)
 	Cross(left, right Rel) (*CrossRel, error)
 	// FetchRemap constructs a fetch relation providing an offset (skipping some
 	// number of rows) and a count (restricting output to a maximum number of
 	// rows).  If count is FETCH_COUNT_ALL_RECORDS (-1) all records will be
 	// returned.  Similar to Fetch but allows for reordering and restricting the
 	// returned columns.
+	//
+	// Deprecated: Use Fetch(...).ChangeMapping() instead.
 	FetchRemap(input Rel, offset, count int64, remap []int32) (*FetchRel, error)
 	// Fetch constructs a fetch relation providing an offset (skipping some number of
 	// rows) and a count (restricting output to a maximum number of rows).  If count
 	// is FETCH_COUNT_ALL_RECORDS (-1) all records will be returned.
 	Fetch(input Rel, offset, count int64) (*FetchRel, error)
+	// Deprecated: Use Filter(...).ChangeMapping() instead.
 	FilterRemap(input Rel, condition expr.Expression, remap []int32) (*FilterRel, error)
 	Filter(input Rel, condition expr.Expression) (*FilterRel, error)
+	// Deprecated: Use JoinAndFilter(...).ChangeMapping() instead.
 	JoinAndFilterRemap(left, right Rel, condition, postJoinFilter expr.Expression, joinType JoinType, remap []int32) (*JoinRel, error)
+	// Deprecated: Use Fetch(...).ChangeMapping() instead.
 	JoinAndFilter(left, right Rel, condition, postJoinFilter expr.Expression, joinType JoinType) (*JoinRel, error)
+	// Deprecated: Use Join(...).ChangeMapping() instead.
 	JoinRemap(left, right Rel, condition expr.Expression, joinType JoinType, remap []int32) (*JoinRel, error)
 	Join(left, right Rel, condition expr.Expression, joinType JoinType) (*JoinRel, error)
+	// Deprecated: Use NamedScan(...).ChangeMapping() instead.
 	NamedScanRemap(tableName []string, schema types.NamedStruct, remap []int32) (*NamedTableReadRel, error)
 	NamedScan(tableName []string, schema types.NamedStruct) *NamedTableReadRel
+	// Deprecated: Use NamedWriteMap(...).ChangeMapping() instead.
 	NamedWriteRemap(input Rel, op WriteOp, tableName []string, schema types.NamedStruct, remap []int32) (*NamedTableWriteRel, error)
```

Comment:
Looks like deprecation message is wrong and NamedWrite() is missing.

Reply:
fixed.